server/web_ui/admin/pkg/LICENSE.md

```diff
@@ -0,0 +1,327 @@
+# Mozilla Public License Version 2.0
+
```

Comment:
Does this need to be copied to these folders or can we link to the root license?